roles/openshift_master/tasks/main.yml

```diff
@@ -131,6 +131,23 @@
 - name: Install the systemd units
   include: systemd_units.yml
 
+- name: Checking for journald.conf
+  stat: path=/etc/systemd/journald.conf
+  register: journald_conf_file
+
+- name: Increase rate limits in journald
+  lineinfile:
+    dest: /etc/systemd/journald.conf
+    regexp: "{{ item.regexp }}"
+    line: "{{ item.line }}"
+  with_items:
+    - { regexp: '^.*RateLimitInterval.*$', line: 'RateLimitInterval=1s' }
+    - { regexp: '^.*RateLimitBurst.*$', line: 'RateLimitBurst=10000' }
```

Comment:
```yaml
- { regexp: '^RateLimitBurst=.*$', line: 'RateLimitBurst=10000' }
```

would be more conservative?

What if the existing values were already higher than the ones we're setting? This could have an unintended effect of doing the opposite of what the task name says.